ast/src/main/java/com/graphicsfuzz/common/ast/type/PushConstantLayoutQualifier.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.graphicsfuzz.common.ast.type;
+
+public class PushConstantLayoutQualifier implements LayoutQualifier {
+
+
+  public PushConstantLayoutQualifier() {
+  }
```

Comment:
This should be unnecessary since it is the default constructor.

Reply:
Done